dora/core/common/src/main/java/alluxio/membership/MembershipManager.java

```diff
@@ -120,7 +120,7 @@ public static MembershipManager create(AlluxioConfiguration conf) {
         case ETCD:
           return EtcdMembershipManager.create(conf);
         case NOOP:
```

Comment:
don't we need to change NOOP to MASTER?
```suggestion
        case MASTER:
```

Reply:
done